src/pages/SignUp/sign-up-schema.js

```diff
@@ -35,8 +33,14 @@ const signUpSchema = yup
         then: yup
           .number()
           .required('Number is required')
+          .test(
+            'len',
+            'Phone number must contain 10 digits',
+            (val) => val.toString().length === 10
+          )
           .typeError('Number is required'),
```

Comment:
Please change all the typeError messages to indicate that the type is wrong instead of saying it's required.

```suggestion
          .typeError('Phone number must be a number),
```